tools/clang/unittests/HLSLExec/ShaderOpArith.xml

```diff
@@ -4184,7 +4184,29 @@ void MSMain(uint GID : SV_GroupIndex,
             const uint32_t OutNum = NUM;
           #endif
 
-          #if IS_UNARY_OP
+          #if TEST_ARRAY_OPERATOR_STATIC_ACCESS
+            uint IndexList[6] = {0, OutNum - 1, 1, OutNum - 2, OutNum / 2, OutNum / 2 + 1};
```

Comment:
The count could be a constant, then used later instead of `6` for clarity.
Plus the `IndexList` could be `const` and formatted to make it easier to see the pattern.
```suggestion
            const uint IndexCount = 6;
            const uint IndexList[IndexCount] = {
                0, OutNum - 1,
                1, OutNum - 2,
                OutNum / 2, OutNum / 2 + 1
            };
```